src/frontend/src/controllers/API/queries/health/use-get-health.ts

```diff
@@ -16,20 +23,44 @@ export const useGetHealthQuery: useQueryFunctionType<
   getHealthResponse
 > = (_, options) => {
   const { query } = UseRequestProcessor();
-
+  const setHealthCheckTimeout = useUtilityStore(
+    (state) => state.setHealthCheckTimeout,
+  );
+  const healthCheckTimeout = useUtilityStore(
+    (state) => state.healthCheckTimeout,
+  );
   /**
    * Fetches the health status of the API.
    *
    * @returns {Promise<AxiosResponse<TransactionsResponse>>} A promise that resolves to an AxiosResponse containing the health status.
    */
   async function getHealthFn() {
-    return (await api.get("/health_check")).data;
-    // Health is the only endpoint that doesn't require /api/v1
+    try {
+      const timeoutPromise = new Promise<never>((_, reject) =>
+        setTimeout(() => reject(createNewError503()), SERVER_HEALTH_INTERVAL),
+      );
+
+      const apiPromise = api.get<{ data: getHealthResponse }>("/health");
```

Comment:
it should be health_check ?

Reply:
we changed to /health, cause health_check was queue more transactions on the database

Reply:
I see. 
Can you add a new line for the /health endpoint here https://github.com/langflow-ai/langflow/blob/7fdfe100985efa7f387637389f5f5930cff7e9b0/docker/frontend/nginx.conf#L20-L22 

Otherwise k8s and docker compose upgrade will be broken

Reply:
yes sure, please check if It is correct in my last commit @nicoloboschi

Reply:
it is, thanks